lib/jekyll-github-metadata/repository.rb

```diff
@@ -119,6 +119,7 @@ def owner_public_repositories
         }
         memoize_value :@owner_public_repositories, Value.new("owner_public_repositories", proc do |c|
           c.list_repos(owner, options).each { |r| r[:releases] = c.releases(r[:full_name]) }
+          c.list_repos(owner, options).each { |r| r[:contributors] = c.contributors(r[:full_name], options) }
```

Comment:
This will make a second request to list repos, so I think you just want to do this to prevent that extra API request:

```suggestion
          c.list_repos(owner, options).each do |r| 
            r[:releases] = c.releases(r[:full_name])
            r[:contributors] = c.contributors(r[:full_name], options)
          end
```

Reply:
I wonder if we can make these `Value`s so they only resolve (make the API request) if they're called. Otherwise we have these 2 (or many more, since auto_paginate is on) extra requests for every repo if any public repos are used in the site.

Reply:
Thanks! Actually I tried this style in first phase as below, but the code I tried doesn't return any key releases or contributors! Thankfully, I think I know a bit more ruby syntax.😭

```ruby
        memoize_value :@owner_public_repositories, Value.new("owner_public_repositories", proc do |c|
          # c.list_repos(owner, options).each { proc do |r|
          #   r[:releases] = c.releases(r[:full_name])
          #   r[:contributors] = c.contributors(r[:full_name], options)
          #   r
          # end }
        end)
```

Reply:
I totally agree. Actually in my case I don't need `release`, but others have tons of repo and contributors and I don't think they want this feature. :sweat_smile:
I will a little more research on what values can be passed in jekyll's `_config.yaml`.